packages/ui/ImageItem/ImageItem.js

```diff
@@ -126,33 +125,29 @@ const ImageItemBase = kind({
 		})
 	},
 
-	render: ({children, css, imageComponent, orientation, placeholder, src, ...rest}) => {
+	render: ({children, css, imageComponent: ImageComponent, orientation, placeholder, src, ...rest}) => {
 		delete rest.selected;
-
 		const isHorizontal = orientation === 'horizontal';
-		const Component = isHorizontal ? Row : Column;
 
 		return (
-			<Component {...rest}>
-				<Cell
+			<div {...rest}>
+				<ImageOverride
```

Comment:
It seems that we don't need the `ImageOverride` anymore.

Please remove the `ImageOverride` and use `<ImageComponent` here.
```
function ImageOverride ({imageComponent, ...rest}) {
	return ComponentOverride({
		component: imageComponent,
		...rest
	});
}
```

Reply:
When I tried to remove this and use 
```{imageComponent} +  className={css.image} placeholder={placeholder} src={src}  ```
it was not possible. because imageComponent is not Component and it's React Element.
So I don't know another way to pass the `className` or other props to `imageComponent` instead of `ImageOverride` .

Reply:
Could you check the following code? It seems to work well to me.

```
	render: ({children, css, imageComponent, orientation, placeholder, src, ...rest}) => {
		...

		return (
			<div {...rest}>
				<ComponentOverride
					className={css.image}
					component={imageComponent}
					placeholder={placeholder}
					shrink={isHorizontal}
					src={src}
				/>
```